test/unit/rabl_test.rb

```diff
@@ -3,14 +3,14 @@
 
 class RablTest < ActiveSupport::TestCase
   test 'render of single template' do
-    @domain = FactoryGirl.build(:domain)
```

Comment:
Because it's only testing the rendering I didn't want to make changes to fit the new permissions so I changed it to `medium` which doesn't have paramteres.